### PR TITLE
Allow volume resizing

### PR DESCRIPTION
**Pull request by r7vme**
Fixes: https://github.com/giantswarm/giantswarm/issues/2861

Requires: https://github.com/giantswarm/k8scloudconfig/pull/336

### CODE REVIEW

File: service/awsconfig/v10/version_bundle.go [hunk 1]
```diff
@@ -10,8 +10,8 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "component",
-				Description: "Add your change here",
+				Component:   "kubernetes",
```
Comment (rossf7) [inline service/awsconfig/v10/version_bundle.go]: Nitpick but could you make the component `cloudconfig`? This matches previous changelogs.

Comment (r7vme) [reply service/awsconfig/v10/version_bundle.go]: done

**Comment by r7vme** [comment]
Thanks, will do.

**Comment by r7vme** [comment]
Tested with upgrade. Merging

**Pull request merged**